### PR TITLE
MIR-924 abstract improvements

### PR DESCRIPTION
**Pull request by yagee-de**
[Link to jira](https://mycore.atlassian.net/browse/MIR-924).

### CODE REVIEW

**Review by fluetze — APPROVED** [review]

**Review by kkrebs — CHANGES_REQUESTED** [review]
Title will be migrated but shown not any longer in metadata page. There goes something wrong?!

File: mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    public void testEscaped() throws JDOMException, IOException {
+        String dataURL = "data:text/xml;charset=UTF-8;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48d"
+            + "Gl0bGVJbmZvIHhtbDpsYW5nPSJlbiI+CiAgICAgICAgICAgIDx0aXRsZT5TdHJ1Y3R1cmFsIEJhc2lzIGZvciBDb21wbGV4IEZv"
+            + "cm1hdGlvbiBiZXR3ZWVuIEh1bWFuIElSU3A1MyBhbmQgdGhlIFRyYW5zbG9jYXRlZCBJbnRpbWluIFJlY2VwdG9yIFRpciBvZiB"
+            + "FbnRlcm9oYWVtb3JyaGFnaWMgJmx0O2kmZ3Q7RS4gY29saSZsdDsvaSZndDsgJmFtcDsgU3RydWN0dXJhbCBDaGFyYWN0ZXJpc2"
+            + "F0aW9uIG9mIEFtb3JmcnV0aW5zIEJvdW5kIHRvIFBQQVIgZ2FtbWE8L3RpdGxlPgogICAgICAgICAgPC90aXRsZUluZm8+";
+        Element test = new Element("test");
+        test.setAttribute("altFormat", dataURL);
+        final Document document = MIRMigration202006Utils.getEmbeddedDocument(new SAXBuilder(), test);
+        XMLOutputter xout = new XMLOutputter(Format.getPrettyFormat());
```
Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java:28]: Codacy found an issue: [Avoid unused local variables such as 'xout'.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867&bid=20036924)

File: mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package org.mycore.mir.migration;
+
+import java.io.ByteArrayInputStream;
```
Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java:3]: Codacy found an issue: [Avoid unused imports such as 'java.io.ByteArrayInputStream'](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java:3]: Codacy found an issue: [Unused import - java.io.ByteArrayInputStream.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867&bid=20036924)

File: mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java (hunk 2 above)
Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java]: Codacy found an issue: [Avoid unused imports such as 'java.io.ByteArrayInputStream'](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

File: mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java (hunk 1 above)
Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java]: Codacy found an issue: [Avoid unused local variables such as 'xout'.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

File: mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java (hunk 2 above)
Comment (yagee-de) [inline mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java]: Codacy found an issue: [Unused import - java.io.ByteArrayInputStream.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867&bid=20036924)

**Review by kkrebs — APPROVED** [review]

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- mir-module/src/test/java/org/mycore/mir/migration/MIRMigration202006UtilsTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6124867&bid=20036924)

**Pull request merged**